src/microbe_stage/CompoundCloudPlane.cs

```diff
@@ -119,7 +119,7 @@ public void UpdatePosition(Int2 newPosition)
     /// <summary>
     ///   Initializes this cloud. cloud2 onwards can be null
     /// </summary>
-    public void Init(FluidSystem fluidSystem, Compound cloud1, Compound? cloud2,
+    public void Init(FluidSystem fluidSystem, int cloudPlaneIndex, Compound cloud1, Compound? cloud2,
```

Comment:
This is still confusingly named. Could you rename this to just `renderPriority` that's at least clear to anyone familiar with godot (transparent) materials.